#service for gettin control_signals from controller

std_msgs/Empty empty
---
last_letter_2/control_signals signals